Language/Haskell/GhcMod/Internal.hs

```diff
@@ -35,14 +45,33 @@ module Language.Haskell.GhcMod.Internal (
   , World
   , getCurrentWorld
   , didWorldChange
+  -- -- * Cabal Helper
+  , getComponents
+  , resolvedComponentsCache
+  , resolveEntrypoint
+  , cached
+  , gmcHomeModuleGraph
+  , ModulePath(..)
+  , GmComponent(..)
+  , GmComponentType(..)
+  , GmModuleGraph(..)
+  , targetGhcOptions
+  , initSession
+  , loadTargets
```

Comment:
Same here. This one really ought to be not exported runGmlT preserves all the internal invariants, i.e. caches up to date and reloading when cached stuff changes.